README.md

```diff
@@ -478,3 +478,14 @@ On CMD run below to get the logstream URL
 curl -dH -X POST WebhookURL
 Browse this url to find the log 
 https://am-project-test-appservice.scm.azurewebsites.net/api/logstream?filter=op:6eaab31a-dc47-4703-8469-1c7410f78ae8,volatile:false
+
+mODULE-13
+poetry add loggly-python-handler
+Set LOG_LEVEL=INFO in env
+Inflask_config set this value-LOG_LEVEL=os.environ.get("LOG_LEVEL") 
+In the create_app function, after you set the logging level- app.logger.setLevel(app.config['LOG_LEVEL']).
+add the lines.
+if app.config['LOGGLY_TOKEN'] is not None:
+ handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app').
+ handler.setFormatter( Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s") ).
+ app.logger.addHandler(handler)
```

Comment:
Why have you got a mix of exercise instructions and Python code in your README? I don't think any of this belongs here.